app/Filament/Admin/Widgets/DiscordPreview.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Filament\Admin\Widgets;
+
+use App\Models\WebhookConfiguration;
+use Filament\Widgets\Widget;
+
+class DiscordPreview extends Widget
+{
+    protected static string $view = 'filament.admin.widgets.discord-preview';
+
+    //Don't invert it, if used normal one will reload the faker every time you write on form.
+    /** @var array<string, string> */
+    protected $listeners = [
+        'widget-refresh' => '$refresh',
+    ];
+
+    protected static bool $isDiscovered = false; // Without this its shown on every Admin Pages
+
+    protected int|string|array $columnSpan = 1;
+
+    public ?WebhookConfiguration $record = null;
+
+    public function getViewData(): array
+    {
+        if (!$this->record || !$this->record->payload) {
+            return [
+                'link' => fn ($href, $child) => $href ? sprintf('<a href="%s" target="_blank" class="link">%s</a>', $href, $child) : $child,
+                'content' => null,
+                'sender' => $this->easterEgg(null),
```

Comment:
```suggestion
                'sender' => ...,
```

We like bird eggs, not bunny eggs 🥚